app/helpers/middleware_server_helper/textual_summary.rb

```diff
@@ -44,4 +44,20 @@ def textual_product
   def textual_version
     @record.properties['Version']
   end
+
+  def textual_lives_on
+    lives_on_ems = @record.try(:lives_on).try(:ext_management_system)
+    return nil if lives_on_ems.nil?
+    lives_on_entity_name = _("Virtual Machine")
+     {
+         :label      => "Underlying #{lives_on_entity_name}",
```

Comment:
I'm probably wrong, but "Underlying" to me implies that the VM is running on Hawkular, and not the other way around. Perhaps "Runs on"? For Middleware, it will always run on an OS, no matter if it's a VM, Container or bare metal.